app/app.js

```diff
@@ -0,0 +1,56 @@
+import {App, IonicApp, Platform} from 'ionic/ionic';
+
+import {HelloIonicPage} from './pages/hello-ionic/hello-ionic';
+import {ListPage} from './pages/list/list';
```

Comment:
First there are some imports defined. 
This is what we use to load other components or services into this root component.